### ISSUE TITLE
Not working with IP camera

### ISSUE THREAD

**Issue by krisstakos** [jneilliii/OctoPrint-BedReady#2]
The plugin looks cool and useful but currently it cannot take snapshot from my IP camera

**Comment by jneilliii** [comment]
Do you have the snapshot url configured properly in the webcam & timelapse settings? Enable debug logging for the plugin and try again and then share your octoprint.log.

**Comment by krisstakos** [comment]
Hm, that was my problem when I tried other CV plugin. Let me try again. Maybe you should put that in the front page of the plugin (You have to configure snapshot url and turn on timelapse in order to work..etc)

**Comment by krisstakos** [comment]
That worked!

**Comment by jneilliii** [comment]
Timelapse doesn't have to be enabled, the snapshot URL just has to be configured. I will update the settings to indicate to the user if it detects an issue with that URL.
